api/api.go

```diff
@@ -919,30 +919,32 @@ func (api *Server) getBlockMetas(start uint64, count uint64) (*iotexapi.GetBlock
 	if start > tipHeight {
 		return nil, status.Error(codes.InvalidArgument, "start height should not exceed tip height")
 	}
-	var res []*iotextypes.BlockMeta
+	var res []*iotextypes.BlockHeaderCore
```

Comment:
Like mentioned in another pr, let’s continue to use blockemeta as the output data structure

Reply:
+1